man-roxygen/path.R

```diff
@@ -1,4 +1,3 @@
-#' @param path This is required The path to the new folder to create relative to
-#'   root.
+#' @param path Path in the user's Dropbox to save the file.
```

Comment:
A lot of these don't _save_ anything; this only really applies to `drop_upload` the way it's written, but that doesn't actually use this template.

Maybe our uses of `path` are too different to template? Otherwise we could do something super general like "path in Dropbox, relative to root"